### ISSUE TITLE
Does not work with dexterity behavior fields 

### ISSUE THREAD

**Issue by kaerumy** [collective/plone.jsonapi.routes#9]
When using Dublic Core Behaviour, fields such as title, description are not found when creating or updating.

```
2014-04-14 08:59:55 INFO plone.jsonapi.routes update_object_with_data::processing key=u'description', value=u'Hello World', field=None
2014-04-14 08:59:55 INFO plone.jsonapi.routes update_object_with_data::skipping key=u'description' 
```

**Comment by ktarasz** [comment]
Maybe we can use something like the that?
```
from plone.dexterity.utils import iterSchemata
        
self.keys = []
for schema in iterSchemata(context):
    self.keys += schema.names()
```

**Comment by ramonski** [comment]
implemented in 0.7
